index.html

```diff
@@ -5,6 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
   </head>
-  <body></body>
+  <body>
+    <nav class="navbar">
+      <div class="navbar__logo">Ahmed</div>
+      <div class="navbar__hamburger">
+        <img src="images/Union.svg" alt="menu" />
+      </div>
+    </nav>
+
+    <header class="header">
```

Comment:
- [ ] The Headline container is meant to be a section tag. This allows your code to be readable and of-course placing the right tags in the right place 🚀 

- [ ] I also think it would be nice if you aligned content in this section with `flexbox`. The importance is that flexbox aligns items with little effort in a right way. Please do try it out 🙏🏻.